cypress/fixtures/constants.ts

```diff
@@ -61,3 +61,17 @@ export const MOCK_REVIEW_DESCRIPTION = "Some interesting description"
 
 export const FORBIDDEN_CHARACTERS = "!@#$%^&*()"
 export const NON_ENGLISH_CHARACTERS = "文கி"
+
+export const timings = {
+  outOfTheWay: 0.2,
+  // greater than the out of the way time
+  // so that when the drop ends everything will
+  // have to be out of the way
+  minDropTime: 0.33,
+  maxDropTime: 0.55,
```

Comment:
Couldn't find usage of these two constants, will they be used in future test? 
Same question, how did we come up with the values here?

Reply:
read the test spec for `react-beautiful-dnd` (see [here](https://github.com/atlassian/react-beautiful-dnd/blob/master/cypress/integration/reorder.spec.js)) and took these values